scripts/install_lbann.sh

```diff
@@ -262,23 +262,23 @@ ${STD_PACKAGES}
     aluminum:
       buildable: true
       version:
-      - 0.4.0
+      - master
```

Comment:
Do we rely on commits to aluminum since release v0.5.0?  If not, would `0.5.0` be more appropriate here?

Reply:
Yes we do unfortunately.